locales/en/messages.json

```diff
@@ -3490,17 +3490,17 @@
         "message": "LEDs without wire ordering number will not be saved.",
         "description": "Message in the wiring modes in Led Strip"
     },
-    "ledStripVtxFunction": {
+    "ledStripLarsonOverlay": {
         "message": "Larson scanner"
     },
-    "ledStripBlinkTitle": {
-        "message": "Blink"
-    },
     "ledStripBlinkAlwaysOverlay": {
         "message": "Blink always"
     },
-    "ledStripBlinkLandingOverlay": {
-        "message": "Blink on landing"
+    "ledStripModifierTitle":{
+        "message": "Modifiers"
+    },
+    "ledStripRainbowOverlay": {
+        "message": "Rainbow"
     },
```

Comment:
Add the same description tag that commented before, please.

Reply:
Wdym?

Reply:
> Add the same description tag that commented before, please.

I think it's pretty obvious what this string for, don't know what should I write in the description.

Reply:
If you see only ”Rainbow" without any other context you know what is it for? That is what see the translators. They don't know this is for LED feature. For this reason I suggested in the other change something generic like "One of the modes for the LED strip feature". This helps a lot the translators to know where the string will be used.

Reply:
Done, I added description in the English file.